chips/earlgrey/src/chip.rs

```diff
@@ -396,11 +396,8 @@ pub extern "C" fn _start_trap_vectored() -> ! {
             j _start_trap
             j _start_trap
             j _start_trap
-        "
-        :
-        :
-        :
-        : "volatile");
-        unreachable_unchecked()
```

Comment:
Did you intend to remove this?

Reply:
yes. Applying the `noreturn` option should have the same effect -- it tells the compiler that we can never fall out of the asm block.